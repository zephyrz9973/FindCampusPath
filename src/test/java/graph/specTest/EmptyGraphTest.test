#This is to test the behavior on empty graph

#Create a graph
CreateGraph graph1

#List nodes of the graph
ListNodes graph1
