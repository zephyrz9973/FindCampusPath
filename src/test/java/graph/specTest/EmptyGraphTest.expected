#This is to test the behavior on empty graph

#Create a graph
created graph graph1

#List nodes of the graph
graph1 contains: